dev-libs/rccl/rccl-5.1.3.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ROCM_VERSION=${PV}
+
+inherit cmake edo rocm
+
+DESCRIPTION="ROCm Communication Collectives Library (RCCL)"
+HOMEPAGE="https://github.com/ROCmSoftwarePlatform/rccl"
+SRC_URI="https://github.com/ROCmSoftwarePlatform/rccl/archive/rocm-${PV}.tar.gz -> rccl-${PV}.tar.gz"
+
+LICENSE="BSD"
+KEYWORDS="~amd64"
+SLOT="0/$(ver_cut 1-2)"
+IUSE="test"
+
+RDEPEND="dev-util/hip
```

Comment:
Explain in your commit message that there is no need to lock hip version.

```
-RDEPEND="dev-util/hip:${SLOT}"                                                                           
+RDEPEND="dev-util/hip         
+dev-util/rocm-smi:${SLOT}"
 DEPEND="${RDEPEND}"    
 BDEPEND=">=dev-util/cmake-3.22                                                                           
 ```

Reply:
Done